src/Raven.Client/Documents/Indexes/IndexDefinition.cs

```diff
@@ -622,7 +628,8 @@ public enum IndexDefinitionCompareDifferences
         DeploymentMode = 1 << 12,
         CompoundFields = 1 << 13,
         ArchivedDataProcessingBehavior = 1 << 14,
+        IndexResetMode = 1 << 15,
```

Comment:
Do you need it? I don't see any usage of this value. Please note that this is about `IndexDefinitionCompareDifferences` - did you add anything to `IndexDefinition`?

Reply:
No, we don't need this, I removed it